packages/components/forms/src/controlled-input-field/ControlledInputField.styles.ts

```diff
@@ -0,0 +1,19 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+
+export const styles = {
+  rootOrLabelDisabled: css({
```

Comment:
This also needs to be applied to the label within ControlledInput to have the exact same styles as before (which can be seen in chromatic). PR to enable passing styles down to the label in ControlledInput is upcoming.

Reply:
@denkristoffer @fabe feel free to review [this mini PR](https://github.com/contentful/forma-36/pull/977) (which blocks this one)